src/docs/product/profiling/getting-started.mdx

```diff
@@ -26,3 +26,4 @@ Profiling depends on Sentry's performance monitoring product being enabled befor
 - Mobile
   - [Android](/platforms/android/profiling/)
   - [iOS](/platforms/apple/guides/ios/profiling/)
+  - [Python](/platforms/python/profiling/)
```

Comment:
Damn, @Zylphrex whoever merges first is going to have to solve a merge conflict here haha

Reply:
I'd recommend not splitting up this list until there are at least a couple bullet points under each of these headings.